packages/amazonq/package.json

```diff
@@ -605,7 +605,7 @@
                 "command": "aws.amazonq.rejectCodeSuggestion",
                 "key": "escape",
                 "mac": "escape",
-                "when": "inlineSuggestionVisible && !editorReadonly && aws.codewhisperer.connected"
+                "when": "inlineSuggestionVisible && !editorReadonly && aws.codewhisperer.connected && aws.codewhisperer.inlineCompletionActive"
```

Comment:
why isn't `inlineSuggestionVisible` sufficient? That would be the main callout I am interested in seeing in the PR description.

Reply:
Because when you ctrl + z, the inline completion item is recovered from VS Code internal stack, but this item is not within our code path's control. It is not a inline completion item generated by our code path, rather, it is a suggestion memorized by VS Code. 

Therefore, when such suggestion is visible, inlineSuggestionVisible is true, but our commands does not work for such suggestion. 

Root cause is : VS Code copies the completion item and put it in the undo redo stack.